src/aws-fsx/ontap-file-system.ts

```diff
@@ -0,0 +1,520 @@
+ *
+ * @see https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-fsx-filesystem-ontapconfiguration.html
+ */
+export interface OntapConfiguration {
+  /**
+   * The number of days to retain automatic backups.
+   * Setting this property to 0 disables automatic backups.
+   * You can retain automatic backups for a maximum of 90 days.
+   *
+   * @default - disable automatic backups
+   */
+  readonly automaticBackupRetention?: Duration;
+
+  /**
+   * Start time for 30-minute daily automatic backup window in Coordinated Universal Time (UTC).
+   *
+   * @default - no backup window
+   */
+  readonly dailyAutomaticBackupStartTime?: aws_fsx.DailyAutomaticBackupStartTime;
+
+  /**
+   * The FSx for ONTAP file system deployment type to use in creating the file system.
+   */
+  readonly deploymentType: OntapDeploymentType;
+
+  /**
+   * The total number of SSD IOPS provisioned for the file system.
+   *
+   * The minimum and maximum values for this property depend on the value of HAPairs and StorageCapacity.
+   * The minimum value is calculated as StorageCapacity * 3 * HAPairs (3 IOPS per GB of StorageCapacity).
+   * The maximum value is calculated as 200,000 * HAPairs.
+   *
+   * @default - 3 IOPS per GB of storage capacity
+   */
+  readonly diskIops?: number;
+
+  /**
+   * The IP address range in which the endpoints to access your file system will be created.
+   *
+   * You can have overlapping endpoint IP addresses for file systems deployed in the same VPC/route tables, as long as they don't overlap with any subnet.
+   *
+   * @default - an unused IP address range from the 198.19.* range
+   */
+  readonly endpointIpAddressRange?: string;
+
+  /**
+   * The ONTAP administrative password for the `fsxadmin` user with which you administer your file system using the NetApp ONTAP CLI and REST API.
+   *
+   * If you don't specify a password, Amazon FSx will not set one. In that case, the user will not be able to log in.
+   *
+   * You can change the admin password at any time through the management console.
+   *
+   * @default - do not set an admin password
+   */
+  readonly fsxAdminPassword?: string;
+
+  /**
+   * How many high-availability (HA) pairs of file servers will power your file system.
+   * First-generation file systems are powered by 1 HA pair.
+   * Second-generation multi-AZ file systems are powered by 1 HA pair.
+   * Second generation single-AZ file systems are powered by up to 12 HA pairs.
+   *
+   * The value of this property affects the values of StorageCapacity, Iops, and ThroughputCapacity.
+   *
+   * Block storage protocol support (iSCSI and NVMe over TCP) is disabled on file systems with more than 6 HA pairs.
+   *
+   * @see https://docs.aws.amazon.com/fsx/latest/ONTAPGuide/administering-file-systems.html#HA-pairs
+   * @see https://docs.aws.amazon.com/fsx/latest/ONTAPGuide/supported-fsx-clients.html#using-block-storage
+   *
+   * @default - 1 HA pair
```

Comment:
`1 HA pair` is different from 1 ? In other words, why not `@default 1`?

Reply:
There's no deep meaning! I've fixed it to 1.

Reply:
Really? Did you push your commit?

Reply:
I could confirm it!